Xamarin.Forms.Core/PoppedToRootEventArgs.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Xamarin.Forms
+{
+	public class PoppedToRootEventArgs : NavigationEventArgs
+	{
+		public PoppedToRootEventArgs(Page page) : base(page)
+		{
+		}
+
+        public IEnumerable<Page> PoppedPages { get; set; }
```

Comment:
The setter should be `private`, and the value of the property passed by constructor argument